tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -157,4 +157,11 @@ Feature: create auth-app token
     And the content in the response should include the following content:
       """
       error parsing expiry. Use e.g. 30m or 72h
-      """
\ No newline at end of file
+      """
+
+  @env-config
+  Scenario: try to create auth-app token with normal user
+    Given user "Brian" has been created with default attributes
+    And the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
```

Comment:
This PR adds test for trying to create auth-app token for other user without impersonation env.
```suggestion
    And the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
```
description and scenario seems opposite to eachother @nirajacharya2 .